test/extended/testdata/build-extended/bc-scripts-by-url.json

```diff
@@ -0,0 +1,39 @@
+{
+  "kind": "BuildConfig",
+  "apiVersion": "v1",
+  "metadata": {
+    "name": "java-extended-build-from-url"
+  },
+  "spec": {
+    "source": {
+      "git": {
+        "uri": "https://github.com/php-coder/java-maven-hello-world.git"
```

Comment:
@bparees @PI-Victor I don't like using personal repos for things like this, but I didn't find a better way.